<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessingWindow</class>
 <widget class="QWidget" name="ProcessingWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>150</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Processing</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../res/Res.qrc">
    <normaloff>:/images/images/application-icon.png</normaloff>:/images/images/application-icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgb(230, 230, 230);
}
</string>
  </property>
  <widget class="QFrame" name="frame1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>430</width>
     <height>70</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
	background-color: rgb(190, 190, 190);
	
	border: 1px solid  rgb(0, 0, 0);
	border-radius: 2px;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
   <property name="midLineWidth">
    <number>2</number>
   </property>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>410</width>
      <height>50</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="messageLabel">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>20</y>
      <width>400</width>
      <height>30</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 75 20pt &quot;Liberation Sans&quot;;
	color: rgb(20, 20, 20);

	background-color: rgba(255, 255, 255, 0);
	
	border: none;
}
</string>
    </property>
    <property name="text">
     <string>PROCESSING</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>430</width>
     <height>50</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
	background-color: rgb(190, 190, 190);
	
	border: 1px solid  rgb(0, 0, 0);
	border-radius: 2px;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
   <property name="midLineWidth">
    <number>2</number>
   </property>
   <widget class="QPushButton" name="exitButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>10</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 75 12pt &quot;Liberation Sans&quot;;
	color: rgb(255, 255, 255);

	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67,  stop: 0 rgb(0, 180, 0), stop: 1 rgb(0, 80, 0));

	border: 1px solid  rgb(0, 0, 0);
	border-radius: 2px;
}

QPushButton:hover {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 0.67,  stop: 0 rgb(0, 200, 0), stop: 1 rgb(0, 110, 0));
}

QPushButton:disabled {
	background-color: rgb(150, 180, 150);
}
</string>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../res/Res.qrc"/>
 </resources>
 <connections/>
</ui>
